etl/steps/data/meadow/animal_welfare/2024-09-13/fur_laws.py

```diff
@@ -0,0 +1,31 @@
+"""Load a snapshot and create a meadow dataset.
+
+"""
+
+from etl.helpers import PathFinder, create_dataset
+
+# Get paths and naming conventions for current step.
+paths = PathFinder(__file__)
+
+
+def run(dest_dir: str) -> None:
+    #
+    # Load inputs.
+    #
+    # Retrieve snapshot and read data.
+    snap = paths.load_snapshot("fur_laws.csv")
+    tb = snap.read()
```

Comment:
you could use the one-liner

```suggestion
    tb = paths.read_snap_table("fur_laws")
```